### DISCUSSION TITLE
The position of mathematical formulation tag

### DISCUSSION THREAD

**Discussion by AaronLuo14** [marp-team/marp#124] [Q&A] [answered]
The tag of a mathematic formula covers the formula. Can we solve it ?
Example: The formula is 
$$
x^n+y^n=z^n \tag{1}
$$
The preview is
![image](https://user-images.githubusercontent.com/44838716/119939205-101e9d80-bfc0-11eb-86d0-a6b1b5e7f06d.png)

**Comment by yhatt** [comment, marked as answer]
It looks like caused due to auto scaling of KaTeX math block provided by Marp Core. How to align tag in KaTeX seems to incompatible with scale-down approach of Marp 😢  (https://github.com/marp-team/marp-core/issues/236)

You can switch math renderer from KaTeX to MathJax. If you are using Marp for VS Code, you can change renderer option in "Markdown > Marp: Math Typesetting".

https://marp.app/docs/guide/math-typesetting#mathjax

**Reply by AaronLuo14** [reply]
Thank you, MathJax works
